client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -57,15 +57,15 @@ const PhpMyAdminCard = ( { translate, siteId, token, loading, disabled } ) => {
 				<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
 				<p>
 					{ translate(
-						'Manage your databases with PHPMyAdmin and run a wide range of operations with MySQL.'
+						'Manage your databases with phpMyAdmin and run a wide range of operations with MySQL.'
 					) }
 				</p>
 				<Button
 					onClick={ () => requestPmaLink( siteId ) }
 					busy={ ! disabled && loading }
 					disabled={ disabled }
 				>
-					<span>{ translate( 'Open PHPMyAdmin' ) }</span>
+					<span>{ translate( 'Open phpMyAdmin' ) }</span>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.